neal/sampler.py

```diff
@@ -235,13 +235,10 @@ def interrupt_function():
         num_betas = int(math.ceil(sweeps / sweeps_per_beta))
         if beta_schedule_type == "linear":
             # interpolate a linear beta schedule
-            beta_step = (beta_range[1] - beta_range[0]) / float(num_betas)
-            beta_schedule = [beta_range[0] + s * beta_step
-                             for s in range(num_betas)]
+            beta_schedule = np.linspace(*beta_range, num=num_betas)
```

Comment:
Just wanted to point out that the linear schedule is going from a beta range that did not include `beta_range[1]` to one that does. i.e. `beta_schedule` was [`beta_range[0]`, `beta_range[1]`), is now [`beta_range[0]`,`beta_range[1]` ].

I like the new version of the code more (where we include `beta_range[1]` in the schedule) as it's more intuitive. However, I needed to point this out in case we need to worry about backwards compatibility.

Reply:
Yep, that's correct. I agree that the end beta should be included in the schedule. I highly doubt this will matter for compatibility so I think we should just change it to the better default of including it.